projects/eval-adxl38x/src/platform/maxim/parameters.h

```diff
@@ -0,0 +1,130 @@
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ *  - Neither the name of Analog Devices, Inc. nor the names of its
+ *    contributors may be used to endorse or promote products derived
+ *    from this software without specific prior written permission.
+ *  - The use of this software may or may not infringe the patent rights
+ *    of one or more patent holders.  This license does not release you
+ *    from the requirement that you obtain separate licenses from these
+ *    patent holders to use this software.
+ *  - Use of the software either in source or binary form, must be run
+ *    on or directly connected to an Analog Devices Inc. component.
+ *
+ * THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT,
+ * MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+ * IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, INTELLECTUAL PROPERTY RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*******************************************************************************/
+#ifndef __PARAMETERS_H__
+#define __PARAMETERS_H__
+
+/******************************************************************************/
+/***************************** Include Files **********************************/
+/******************************************************************************/
+#include "maxim_irq.h"
+#include "maxim_spi.h"
+#include "maxim_gpio.h"
+#include "maxim_uart.h"
+#include "maxim_uart_stdio.h"
+
+/******************************************************************************/
+/********************** Macros and Constants Definitions **********************/
+/******************************************************************************/
+
+#if (TARGET_NUM == 32690)
+#define UART_IRQ_ID     UART2_IRQn
+#define UART_DEVICE_ID  2
+#define UART_BAUDRATE   57600
+#else
+#define UART_IRQ_ID     UART0_IRQn
+#define UART_DEVICE_ID  0
+#define UART_BAUDRATE   57600
+#endif
+#define UART_EXTRA      &adxl38x_uart_extra_ip
+#define UART_OPS        &max_uart_ops
+
+#if (TARGET_NUM == 78000)
+#define SPI_DEVICE_ID   1
+#define SPI_CS          1
+#elif (TARGET_NUM == 32650)
+#define SPI_DEVICE_ID   1
+#define SPI_CS          0
+#elif (TARGET_NUM == 32690)
+#define SPI_DEVICE_ID   0
+#define SPI_CS          1
+#else
+#define SPI_DEVICE_ID   0
+#define SPI_CS          0
+#endif
```

Comment:
so builds.json contains the build only for max78000... I would only leave the platform on which you have actually tested it here, I suppose it is max78000, right?

Reply:
Yes, it was tested only on max78000.